cfgov/tccp/jinja2/tccp/landing_page.html

```diff
@@ -60,25 +61,28 @@ <h2>
         </h2>
         <ul class="m-list">
             <li class="m-list__item">
-                Larger pool of cards, likely including banks you’ve never heard of with great rates
+                No paid advertising.
             </li>
             <li class="m-list__item">
+                Interest rates are rated, to help you compare.
+            </li>
+            <li class="m-list__item">
+                This tool is updated every six months and is currently based on data from {{ date(stats.first_report_date) }}. Advertised purchase APRs change over time. Check rates before applying and
```

Comment:
We should handle the case when there are no cards in the database for whatever reason.

```suggestion
                This tool is updated every six months
                {%- if stats.first_report_date %}
                and is currently based on data from {{ date(stats.first_report_date) }}
                {%- endif %}.
                Advertised purchase APRs change over time. Check rates before applying and
```